features/netsocket/mbed_lib.json

```diff
@@ -2,6 +2,22 @@
     "name": "nsapi",
     "config": {
         "present": 1,
-        "default-stack": "LWIP"
+        "default-stack": "LWIP",
+        "dns-response-wait-time": {
+            "help": "How long the DNS translator waits for a reply from a server",
+            "value": 5000
+        },
+        "dns-total-retries": {
```

Comment:
Either should be `attempts`, or implementation should treat it as `retries`

Reply:
Changed to attempts.